pkg/scaffold/olm-catalog/csv_test.go

```diff
@@ -51,24 +49,29 @@ func TestCSVNew(t *testing.T) {
 	csvVer := "0.1.0"
 	projectName := "app-operator"
 
-	err := s.Execute(&input.Config{ProjectName: projectName},
-		&CSV{CSVVersion: csvVer, pathPrefix: testDataDir},
-	)
+	sc := &CSV{CSVVersion: csvVer, pathPrefix: testDataDir}
+	err := s.Execute(&input.Config{ProjectName: projectName}, sc)
 	if err != nil {
 		t.Fatalf("Failed to execute the scaffold: (%v)", err)
 	}
 
+	csvStr := buf.String()
+	tsSplit := regexp.MustCompile("creationTimestamp: \".+\"").Split(csvStr, 2)
+	if len(tsSplit) == 2 {
+		csvStr = tsSplit[0] + `creationTimestamp: "2019-01-01T00:00:00Z"` + tsSplit[1]
+	} else {
+		t.Errorf("Expected to find creationTimestamp in CSV metadata, found none")
+	}
+
 	// Get the expected CSV manifest from test data dir.
-	csvFileName := getCSVFileName(projectName, csvVer)
-	csvPath := filepath.Join(testOLMDir, projectName, csvVer, csvFileName)
-	csvExpBytes, err := afero.ReadFile(s.Fs, csvPath)
+	csvExpBytes, err := afero.ReadFile(s.Fs, sc.getCSVPath(csvVer))
 	if err != nil {
 		t.Fatal(err)
 	}
 	csvExp := string(csvExpBytes)
-
-	if csvExp != buf.String() {
+	if csvExp != csvStr {
 		diffs := diffutil.Diff(csvExp, buf.String())
+		// If the only difference is in the timestamp, pass the test.
```

Comment:
Maybe it would be better to put this comment ~line 59, where the `creationTimestamp` gets fixed? Otherwise it seems out of place, since at this point in the code you're doing an exact check of `csvStr` against `csvExp`.